### ISSUE TITLE
Lambda runtime segments appearing in traces

### ISSUE THREAD

**Issue by bonybrown** [aws/aws-xray-sdk-ruby#38]
Adding x-ray instrumentation to a lambda causes segments in the trace that are due to the lambda runtime, like this:

![image](https://user-images.githubusercontent.com/1063621/69515924-3830d900-0fa4-11ea-9240-c04c1308e1c9.png)

These shouldn't appear, and should be filtered out by `XRay::NetHttp::HTTPInstanceInterceptor#lambda_runtime_request?` but the current implementation is not correct. 

A correct implementation is below (can be used as a monkey-patch to fix the current sdk release)

```ruby
module XRay
  module NetHttp
    module HTTPInstanceInterceptor
      def lambda_runtime_request?(_req)
        ENV['AWS_LAMBDA_RUNTIME_API'] == "#{address}:#{port}"
      end
    end
  end
end
```

I'll create a PR for this. Just need the issue to reference.

**Comment by bonybrown** [comment]
PR #39 ready for review.

**Comment by srprash** [comment]
Reopening since the fix is pending release.

**Comment by srprash** [comment]
This issue is fixed in v0.11.4
